packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,646 @@
+        "deploy-preview": {
+          "$ref": "#/definitions/DeployPreviewContext"
+        },
+        "branch-deploy": {
+          "$ref": "#/definitions/BranchDeployContext"
+        }
+      }
+    },
+    "redirects": {
+      "type": "array",
+      "default": [{ "from": "/old-path", "to": "/new-path" }],
+      "items": {
+        "title": "Redirects",
+        "description": "You can manage your redirects directly in your `netlify.toml`. For each redirect you want to declare, add an entry with the [[redirects]] heading.",
+        "type": "object",
+        "default": { "from": "/old-path", "to": "/new-path" },
+        "properties": {
+          "from": {
+            "title": "From",
+            "description": "The path you want to redirect.",
+            "type": "string",
+            "default": "/old-path"
+          },
+          "to": {
+            "title": "To",
+            "description": "The URL or path you want to redirect to.",
+            "type": "string",
+            "default": "/new-path"
+          },
+          "status": {
+            "title": "HTTP status code",
+            "description": "The HTTP status code you want to use in that redirect; 301 by default.",
+            "type": "integer",
+            "default": 301,
+            "minimum": 100,
+            "exclusiveMaximum": 600
+          },
+          "force": {
+            "title": "Force",
+            "description": "Whether to override any existing content in the path or not; false by default.",
+            "type": "boolean",
+            "default": true,
+            "x-taplo": {
+              "docs": {
+                "main": "Whether to override any existing content in the path or not; false by default. Visit the [shadowing](https://docs.netlify.com/routing/redirects/rewrites-proxies/#shadowing) instructions for more details."
+              },
+              "links": {
+                "key": "https://docs.netlify.com/routing/redirects/rewrites-proxies/#shadowing"
+              }
+            }
+          },
+          "query": {
+            "title": "Query string parameters",
+            "description": "Query string parameters REQUIRED to match the redirect.",
+            "type": "object",
+            "additionalProperties": {
+              "type": "string"
+            },
+            "x-taplo": {
+              "docs": {
+                "main": "Query string parameters REQUIRED to match the redirect. Visit the [query parameters](https://docs.netlify.com/routing/redirects/redirect-options/#query-parameters) instructions for more details."
+              },
+              "links": {
+                "key": "https://docs.netlify.com/routing/redirects/redirect-options/#query-parameters"
+              }
+            }
+          },
+          "conditions": {
+            "title": "Conditions",
+            "description": "Redirect based on conditions including browser language, geolocation, identity role, and/or cookie presence.",
```

Minor edit to follow our style guide

```suggestion
            "description": "Redirect based on conditions including browser language, geolocation, identity role, or cookie presence.",
```